__tests__/endpoints.test.js

```diff
@@ -145,4 +145,76 @@ describe('GET /api/reviews/:review_id/comments', () => {
             expect(result.body).toEqual({message: 'Resource not found' })
         })
     });
-});
\ No newline at end of file
+});
+describe('POST /api/reviews/:review/comments', () => {
+    test('should respond: 201 with new comment ', () => {
+        //Arrange
+        const body = {
+            username: 'mallionaire',
+            body: 'This is my new comment!'
+          }
+        return request(app).post('/api/reviews/3/comments')
+        .send(body)
+        .expect(201)
+        .then(({body}) => {
+                expect(typeof body.comment.author).toBe('string');
+                expect(typeof body.comment.body).toBe('string');
+                expect(typeof body.comment.comment_id).toBe('number');
+                expect(typeof body.comment.created_at).toBe('string');
+                expect(typeof body.comment.review_id).toBe('number');
+                expect(typeof body.comment.votes).toBe('number');
+        })
+    });
+    test('should respond: 201 with new comment ', () => {
```

Comment:
You said: 'should respond: 201 with new comment', but you're only checking the status code.